### DISCUSSION TITLE
Configure docker to build an image of bentoservice located in a remote minio server

### DISCUSSION THREAD

**Discussion by athulprathap** [bentoml/BentoML#943] [General] [answered]
I have configured a remote Minio server for storing saves BentoService files to ~/minio/aisolrepo/.
Used the following reference command to configure the remote s3 storage.
AWS_ACCESS_KEY_ID=Q3AM3UQ867SPQQA43P2F AWS_SECRET_ACCESS_KEY=zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG AWS_REGION=us-east-1 bentoml yatai-service-start --s3-endpoint-url=https://play.min.io:9000 --repo-base-url=s3://chaoyu-bentoml
Models are getting displaced in yatai UI.After this trying to build the docker image in my local system using the docker file located in remote minio server.
Followed the documentation given in https://github.com/bentoml/BentoML.Made changes in the config file to add the remote gRPC server
Ran "saved_path=$(bentoml get IrisClassifier:latest -q | jq -r ".uri.uri")"
Got 
echo $saved_path
s3://aisolrepo/IrisClassifier/20200720081955_358AFA.tar.gz

After this while trying to run docker build -t iris-classifier $saved_path getting error of 

unable to prepare context: path "s3://aisolrepo/IrisClassifier/20200720081955_358AFA.tar.gz" not found

So my question is "**How do we configure docker to build an image of bentoservice located in a remote minio server**" ?

**Comment by parano** [comment, marked as answer]
Hi @athulprathap 

Assuming you have configured a remote YataiService, you can use the `bentoml retrieve` command to download the BentoService saved bundle directory to any local file directory. E.g:

```bash
$ bentoml retrieve IrisClassifier/20200720081955_358AFA --target_dir=/tmp/irirs-classifier-saved-bundle

$ cd /tmp/irirs-classifier-saved-bundle

$ docker build . 
```

In the next BentoML release, a new CLI command `$ bentoml containerize` will be available and you can use that to build docker image located in a remote YataiServcie directly(it downloads the saved bundle files locally under the hood)

**Reply by athulprathap** [reply]
@parano Thank you!!This is working
